spec/methods/auth/auth.ts

```diff
@@ -2,7 +2,9 @@ import { XOR } from 'ts-xor';
 import { apiBasePath, RootResources } from '../../methods';
 import { ApiResponse, MessageResponse, HttpMethods, MessageResult } from '../../responses';
 import { AuthData, Challenges } from '../../user';
-import { keysAreStrings as keysAreStrings } from '../../util';
+import { keysAreStrings as keysAreStrings,
```

Comment:
VS Code sometimes leaves artifacts like this when you rename a symbol, depending on which file you rename it from.  Wanna remove these `as ...` clauses and just put it on one line?